etip/restful_api/serializers.py

```diff
@@ -23,10 +23,21 @@ class Meta:
           'is_in_exodus',
           'code_signature',
           'network_signature',
+          'api_key_ids',
           'website',
           'maven_repository',
           'group_id',
           'artifact_id',
           'gradle',
+          'created',
+          'updated',
+          'creation_date',
```

Comment:
**question**: Why do we need `created` and `creation_date`? Seems redundant to me.